README.md

```diff
@@ -417,6 +438,50 @@ cargo_generate_version = ">=0.9.0"
 
 The format for the version requirement is [documented here](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html).
 
+## Conditional template settings.
+
+Using `cargo-generate.toml`, values and some [`Rhai`](https://rhai.rs/book/) syntax, the template author can make certain conditional decisions before expansion of the template.
+
+`include`, `exclude`, `ignore` and `placeholders` can all be used in sections that are only used based upon the value of one or more values, possibly input by the user using the interactive prompt (if the values in question are defined as placeholders in the non-conditional section).
+
+Example:
```

Comment:
suggestion: in order to make this whole thing a little easier to digest I would not pack such a dense example, and rather go for maybe 2-4 smaller examples, where every single one would focus on certain things. 

Such big examples require a lot of mental load to process and try to illustrate too many points at the same time as a result new users getting more easily lost. 

This is just a suggestion thou.